packages/js/src/general/components/llm-txt-opt-in-notification.js

```diff
@@ -0,0 +1,54 @@
+import { Toast, Button, useSvgAria, useToggleState } from "@yoast/ui-library";
+import { __ } from "@wordpress/i18n";
+import { ReactComponent as YoastIcon } from "../../../images/Yoast_icon_kader.svg";
+import { ArrowNarrowRightIcon } from "@heroicons/react/outline";
+import { useSelectGeneralPage } from "../hooks";
+import { useCallback } from "@wordpress/element";
+
+/**
+ * Shows only once.
+ *
+ * @returns {JSX.Element} The LLM txt opt-in notification component.
+ */
+export const LlmTxtOptInNotification = () => {
+	const svgAriaProps = useSvgAria();
+	const llmTxtSettingsUrl = useSelectGeneralPage( "selectAdminLink", [],  "?page=wpseo_page_settings#/llms-txt" );
+
+	const [ isVisible, toggleIsVisible, setIsVisible ] = useToggleState( true );
+	const handleShow = useCallback( () => {
+		sessionStorage.setItem( "highlight-setting", "llm-txt" );
```

Comment:
Did we consider any alternative other than using the session storage? I wouldn't introduce this unless it's necessary. Using session storage creates some more aspects to be dealt with, like:
* most importantly, we need to have a yoast specific key to our session storage data, something like `yoast-highlight-setting`
* we need to consider cases where session storage is disabled and maybe also add them in our test instructions. not sure if there needs to be defensive coding added, but we should check those cases (examples of ways to disable session storage can be found in [this PR](https://github.com/Yoast/wordpress-seo/pull/22186)) 
* at the very least, this deserves its own entry in the technical decisions.